### DISCUSSION TITLE
icons not working

### DISCUSSION THREAD

**Discussion by neilbags** [tw93/Pake#612] [Q&A]
What do I need to do to make icons work? I've tried a bunch of different images but I can't get icons in the app menu or in the system tray

**Comment by Tlntin** [comment]
do you use Pake in GitHub actions or locally?

**Reply by neilbags** [reply]
Locally, on Ubuntu generating .debs

**Reply by Tlntin** [reply]
i think you need to change some files.
https://github.com/tw93/Pake/blob/master/src-tauri/tauri.linux.conf.json#L4
https://github.com/tw93/Pake/blob/master/src-tauri/assets/com-tw93-weread.desktop
and the png picture file
https://github.com/tw93/Pake/blob/master/src-tauri/png/weread_512.png
